java/gradle/ice.gradle

```diff
@@ -68,11 +78,17 @@ spotless {
     cleanthat()
 
     // Formatting
-    googleJavaFormat()
+    googleJavaFormat().aosp()
```

Comment:
This is the `Android Open Source Project` sub-style.
It's built off the google style, but uses 4 spaces instead of 2.
It might also do some small changes, but I haven't seen them make a difference in our code.